core/src/navigation/TabNav.html

```diff
@@ -122,16 +124,28 @@
             {/each}
             {:else}
             <li class="fd-nested-list__item" uid="{index}-0">
-              <a
-                href="javascript:void(null)"
-                class="fd-nested-list__link has-child"
-                aria-haspopup="true"
-                aria-expanded="{dropDownStates[key+index]}"
-                aria-selected="{isSelectedCat(key,selectedNodeForTabNav)}"
-                on:click|preventDefault="{() => toggleDropdownState(key+index)}"
-              >
-                <span class="fd-nested-list__title">{$getTranslation(key)}</span>
-              </a>
+              <div class="fd-nested-list__content has-child" tabindex="0">
+                <a
+                  href="javascript:void(null)"
+                  tabindex="-1"
+                  class="fd-nested-list__link"
+                  aria-haspopup="true"
+                  aria-expanded="{dropDownStates[key+index]}"
+                  aria-selected="{isSelectedCat(key,selectedNodeForTabNav)}"
+                  on:click|preventDefault="{() => toggleDropdownState(key+index)}"
+                >
+                  <span class="fd-nested-list__title">{$getTranslation(key)}</span>
+                </a>
+                <a
+                  class="fd-nested-list__expand-icon"
+                  href="#"
+                  tabindex="-1"
+                  aria-controls="EX100L2"
```

Comment:
the same question as for `core/src/navigation/LeftNav.html`. If you need `aria-controls` here, then it should be connected with some `id`. Otherwise, it doesn't make sense to use it here.